### PR TITLE
Add the documentation for installing OTX on XPU devices

### PR DESCRIPTION
**Pull request by kprokofi**
### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: docs/source/guide/get_started/installation.rst [hunk 1]
```diff
@@ -77,6 +78,71 @@ Refer to the `official installation guide <https://pytorch.org/get-started/previ
...
 OpenVINO™ Training Extensions command line functionality.
 
+******************************************************
+Install OpenVINO™ Training Extensions for users (XPU)
+******************************************************
+
+1. Proceed with 1-2 steps from the above instructions
+cloning the repository and setting up the environment
+
+2. install Intel® Extension
```
Comment (eunwoosh) [inline docs/source/guide/get_started/installation.rst]:
```suggestion
2. Install Intel® Extension
```
It's minor one but to keep unity

File: docs/source/guide/get_started/installation.rst [hunk 2]
```diff
@@ -77,6 +78,71 @@ Refer to the `official installation guide <https://pytorch.org/get-started/previ
...
+    tar -zxvf mmcv-full-1.7.0.tar.gz
+    sed -i 's/c++14/c++17/g' mmcv-full-1.7.0/setup.py
+    MMCV_WITH_OPS=1 pip install ./mmcv-full-1.7.0
+    rm -r mmcv-full-1.7.0 mmcv-full-1.7.0.tar.gz
+
+* From source
+
+.. code-block::
+
+    wget https://gist.githubusercontent.com/sovrasov/44c41202c09ab38d657e796fccc86181/raw/146a50c5f99c8721c0bcc0fcc25b19064c4b29a2/mmcv_1_7_0_setup.patch
```
Comment (eunwoosh) [inline docs/source/guide/get_started/installation.rst:118]:
```suggestion
    git clone https://github.com/open-mmlab/mmcv
    cd mmcv
    git checkout v1.7.0
    wget https://gist.githubusercontent.com/sovrasov/44c41202c09ab38d657e796fccc86181/raw/146a50c5f99c8721c0bcc0fcc25b19064c4b29a2/mmcv_1_7_0_setup.patch
```
How about adding this part?

**Review by eunwoosh — COMMENTED** [review]
Thanks for your work :) I left comments. Please take a look.

File: docs/source/guide/get_started/installation.rst (hunk 2 above)
Comment (kprokofi) [reply docs/source/guide/get_started/installation.rst:118]: Sure, thank you

**Review by eunwoosh — APPROVED** [review]

**Review by sungmanc — APPROVED** [review]

**Pull request merged**